stock_inventory_lockdown/models/stock_location.py

```diff
@@ -0,0 +1,31 @@
+# © 2016 Numérigraphe SARL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class StockLocation(models.Model):
+    """Refuse changes during exhaustive Inventories"""
+
+    _inherit = "stock.location"
+    _order = "name"
```

Comment:
@AdriaGForgeFlow @rousseldenis 
Do you know if there is a reason for changing the default order (default is `complete_name`)?
It seems pretty brutal. In my case, it makes tests fail in https://github.com/OCA/stock-logistics-warehouse/pull/997

Reply:
Indeed, I didn't notice it. This is obviously not advised to change _order parameter.

@LoisRForgeFlow @AdriaGForgeFlow Is this needed ?

Reply:
@AdriaGForgeFlow

Reply:
This is coming from the creation of this module in v9. I guess it made sense back then but I'm not really sure it is needed anymore, probably not